src/components/calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
[OPTIONAL ] I suggest you remove comments from your code and also ensure you do not disable linters. Linter checks ensure that you do not make simple mistakes in your code and also that you create code consistent with developer best practices.